examples/join-tables-and-streams/join_tables_and_streams.out

```diff
@@ -0,0 +1,4 @@
+#At the command line, navigate to the directory that contains the 
+#`.bal` file and run the `ballerina run` command. 
```

Comment:
Use the below text as we need to have consistent message for all samples. This is enforced with https://github.com/ballerina-platform/ballerina-lang/pull/12508
# To run this sample, navigate to the directory that contains the
# `.bal` file and issue the `ballerina run` command.

Reply:
done